templates/backup-pre-postgres.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TMPDIR="{{ POSTGRES_SERVER_BACKUP_DIR }}"
+
+rm -rf "$TMPDIR"
+mkdir -p "$TMPDIR"
+sudo -u postgres pg_dumpall > "$TMPDIR/postgresql_all_dbs.sql"
```

Comment:
+1 to splitting - managing/restoring one giant dump for all DBs can be pretty painful.